source/reference/minio-server/settings/object-lambda.rst

```diff
@@ -10,49 +10,193 @@ Object Lambda Function Settings
 
-You can specify multiple webhook endpoints as Lambda targets by appending a unique identifier ``_FUNCTIONNAME`` for each Object Lambda function.
-For example, the following command sets two distinct Object Lambda webhook endpoints:
+You can establish or modify settings by defining:
 
-.. code-block:: shell
-   :class: copyable
+- an *environment variable* on the host system prior to starting or restarting the MinIO Server.
+  Refer to your operating system's documentation for how to define an environment variable.
+- a *configuration setting* using :mc:`mc admin config set`.
 
-   export MINIO_LAMBDA_WEBHOOK_ENABLE_myfunction="on"
-   export MINIO_LAMBDA_WEBHOOK_ENDPOINT_myfunction="http://webhook-1.example.net"
-   export MINIO_LAMBDA_WEBHOOK_ENABLE_yourfunction="on"
-   export MINIO_LAMBDA_WEBHOOK_ENDPOINT_yourfunction="http://webhook-2.example.net"
+If you define both an environment variable and the similar configuration setting, MinIO uses the environment variable value.
+
+Some settings have only an environment variable or a configuration setting, but not both.
 
 .. include:: /includes/common-mc-admin-config.rst
    :start-after: start-minio-settings-test-before-prod
    :end-before: end-minio-settings-test-before-prod
 
-Environment Variables
----------------------
 
-.. envvar:: MINIO_LAMBDA_WEBHOOK_ENABLE
+Enable
+------
+
+.. tab-set::
+
+   .. tab-item:: Environment Variable
+      :sync: envvar
+
+      .. envvar:: MINIO_LAMBDA_WEBHOOK_ENABLE
+
+         Specify ``"on"`` to enable the Object Lambda webhook endpoint for a handler function.
+
+         Requires specifying :envvar:`MINIO_LAMBDA_WEBHOOK_ENDPOINT`.
+
+	 You can specify multiple webhooks as Lambda targets by appending a unique identifier ``_FUNCTIONNAME`` for each Object Lambda function.
+         For example, the following command enables two distinct Object Lambda webhook endpoints:
+
+         .. code-block:: shell
+            :class: copyable
+
+            export MINIO_LAMBDA_WEBHOOK_ENABLE_myfunction="on"
+            export MINIO_LAMBDA_WEBHOOK_ENABLE_yourfunction="on"
+
+   .. tab-item:: Configuration Setting
+      :sync: config
+
+      .. mc-conf:: lambda_webhook enable
+         :delimiter: " "
+
+         *Optional*
+
+         Specify ``"on"`` to enable the Object Lambda webhook endpoint for a handler function.
+	 Requires specifying :mc-conf:`~lambda_webhook.endpoint`.
+
+	 Example:
+
+	 .. code-block:: shell
+	    :class: copyable
+
+            mc admin config set myminio lambda_webhook:myfunction endpoint="https://example.com/" enable=on
+
+Endpoint
+--------
+
+.. tab-set::
+
+   .. tab-item:: Environment Variable
+      :sync: envvar
+
+      .. envvar:: MINIO_LAMBDA_WEBHOOK_ENDPOINT
+
+         The HTTP endpoint of the lambda webhook for the handler function.
+
+         You can specify multiple webhook endpoints as Lambda targets by appending a unique identifier ``_FUNCTIONNAME`` for each Object Lambda function.
```

Comment:
```suggestion
         You can specify multiple webhook endpoints as Lambda targets by appending a unique identifier for each Object Lambda function.
```

I won't do it for any other occurrences, but if you drop it on one, you'll need to drop it throughout.